src/main/R/FOR_plot.R

```diff
@@ -0,0 +1,19 @@
+library(ggplot2)
+
+# load FOR and CGD sets
+falseOmRate <- read.table("/Users/joeri/Desktop/GAVIN-APP/1000G_diag_FDR/FOR/FOR_results_per_gene_r1.0.tsv", sep="\t", header=T)
+cgd <- read.table("/Users/joeri/github/rvcf/src/test/resources/GenesInheritance31aug2016.tsv", sep="\t", header=T)
+
```

Comment:
hardcoded paths, not sure how to fix in R, can those be relative to the project?